"ignoring old recipe for target 'clean'" warning while building KOS

Hopefully this hasn't been reported and decided to be ok before, couldn't find an existing issue about it. Am a novice with makefiles and not sure if there's a way to do something like add something extra that would be done for clean rather than redefining it (since it seems in this case and for the main `/kernel/Makefile` we're doing the existing clean + extra).

```
make -C  kernel
make[1]: Entering directory '/home/quzar/dcdev/KallistiOS/kernel'
Makefile:30: warning: overriding recipe for target 'clean'
/home/quzar/dcdev/KallistiOS/Makefile.prefab:19: warning: ignoring old recipe for target 'clean'
```

```
make -C  kernel
make[4]: Entering directory '/home/quzar/dcdev/KallistiOS/kernel/arch/dreamcast/kernel'
Makefile:50: warning: overriding recipe for target 'clean'
/home/quzar/dcdev/KallistiOS/Makefile.prefab:19: warning: ignoring old recipe for target 'clean'
```

```
make -C  exports
make[2]: Entering directory '/home/quzar/dcdev/KallistiOS/kernel/exports'
Makefile:18: warning: overriding recipe for target 'clean'
/home/quzar/dcdev/KallistiOS/Makefile.prefab:19: warning: ignoring old recipe for target 'clean'
kos-cc   -c nmmgr.c -o nmmgr.o
cp nmmgr.o kernel_exports.o exports.o library.o /home/quzar/dcdev/KallistiOS/kernel/build/
make[2]: Leaving directory '/home/quzar/dcdev/KallistiOS/kernel/exports'
```

--- a/Makefile.prefab
+++ b/Makefile.prefab
@@ -15,8 +15,10 @@ endif
 defaultall: $(OBJS) subdirs
 	-cp $(OBJS) $(KOS_BASE)/$(BUILD_TARGET)/build/
 
-clean: clean_subdirs
+defaultclean: clean_subdirs
 	-rm -f $(OBJS) *.bck
+
+clean: defaultclean
 
 include $(KOS_BASE)/Makefile.rules
 

--- a/kernel/Makefile
+++ b/kernel/Makefile
@@ -26,7 +26,7 @@ stubs/arch_export_stubs.c: arch/$(KOS_ARCH)/exports-$(KOS_SUBARCH).txt
 
 include $(KOS_BASE)/Makefile.prefab
 
-clean: clean_subdirs
+clean: defaultclean
 	rm -f build/*.o
 	rm -f build/libc/*.o
 	rm -f romdisk/*.o

--- a/kernel/exports/Makefile
+++ b/kernel/exports/Makefile
@@ -14,5 +14,5 @@ kernel_exports.o: kernel_exports.c
 kernel_exports.c: ../exports.txt
 	$(KOS_BASE)/utils/genexports/genexports.sh ../exports.txt kernel_exports.c kernel_symtab
 
-clean:
-	-rm -f kernel_exports.c $(OBJS)
+clean: defaultclean
+	-rm -f kernel_exports.c

